Week9Serialization/Week9Serialization/ISerializer.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Week9Serialization
+{
+    /// <summary>
+    /// Defines the ISerializer interface that contains the Serialize() and Deserialize methods;
+    /// </summary>
+    interface ISerializer
+    {
+        /// <summary>
+        /// Defines a method that serializes a list of Persons. 
+        /// </summary>
+        void Serialize<T>(PersonList<T> list) where T :Person;
```

Comment:
The XML commentary for this method could be something like, "When implemented in a derived class, serializes the specified <see cref="PersonList{T}" />." That would make it really easy for someone to understand and use your API.